<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriteSmsScreen</class>
 <widget class="QWidget" name="WriteSmsScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editTo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="editText">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="VirtualKeyboard" name="virtualKeyboard" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSend">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="mobilephone2.qrc">
         <normaloff>:/images/sms-send.svg</normaloff>:/images/sms-send.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonHome">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="mobilephone2.qrc">
         <normaloff>:/images/home.svg</normaloff>:/images/home.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VirtualKeyboard</class>
   <extends>QWidget</extends>
   <header>virtualkeyboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mobilephone2.qrc"/>
 </resources>
 <connections/>
</ui>
